plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/KuduQueryRunnerFactory.java

```diff
@@ -152,4 +154,19 @@ public static Session createSession(String schema)
                 .setSchema(schema)
                 .build();
     }
+
+    public static void main(String[] args)
```

Comment:
This is just for manual testing correct?

Reply:
Yes. Also this is where log-manager gets pulled in because of `Logging.initialize`.